test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by Git. 
```

Comment:
Great job @Bludidi 🤝

- [ ] Kindly add this file `test.md` to the `.gitignore file` so that it can be ignored by git. As of now it is appearing on git meaning it was not ignored.

Reply:
I have edited the file name I had put earlier to test.md